cosmosdb_emulator/README.md

```diff
@@ -0,0 +1,88 @@
+# Time Tracker CLI
+
+Here you can find all the source code of the Time Tracker CLI. 
+This is responsible for automatically generating fake data for the Cosmos emulator, 
+in order to have information when testing new features or correcting bugs.
+
+## Prerequisites
+
+- Backend and cosmos emulator containers up.
+- Environment variables correctly configured
+
+### Environment Variables.
+
+The main environment variables that you need to take into account are the following:
+
+```shell
+export DATABASE_ACCOUNT_URI=https://azurecosmosemulator:8081
+export DATABASE_MASTER_KEY=C2y6yDjf5/R+ob0N8A7Cgv30VRDJIWEHLM+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw==
+export DATABASE_NAME=time_tracker_testing_database
+```
+Verify that the variables are the same as those shown above. 
+
+## How to use Time Tracker CLI?
+
+If we are in the project's root folder, we need to redirect to the folder `cosmosdb_emulator` and open a terminal.
+
+We have two main alternatives for running the CLI:
+
+### Execute CLI with flags.
+   
+In order to see all the available flags for the CLI we are going to execute the following command:
+
+```shell
+./cli.sh main.py --help
+```
+
+When executing the above command, the following information will be displayed:
+
+![image](https://user-images.githubusercontent.com/56373098/127604274-041c2af7-d7a8-4b8d-b784-8280773b68c8.png)
+
+Where you can see the actions we can perform on a given Entity:
+
+Currently, the CLI only allows the creation of Time-entries and allows the deletion of any entity.
+
+Available Actions:
+
+- Create: Allows creating new fake data about a certain entity.
+- Delete: Allows creating cosmos data about a certain entity.
```

Comment:
In the Delete action you wrote: "Allows creating....". In this case it should say "Allows deleting....".

Reply:
Thank you Edgar, I overlooked it, solved 💯